solutions/bronze/cf-1914C.mdx

```diff
@@ -0,0 +1,51 @@
+---
+id: cf-1914C
+source: CF
+title: Quests
+author: Chu Minjae
+---
+
+## Implementation
+
+We can find the maximum experience point available by comparing the largest available experience point gained by completing one more quest or not.
+
+**Time Complexity:** $\mathcal{O}(t \cdot \textbf{limit})$, where $t$ is the number of test cases and $\textbf{limit}$ as the farthest location of quest.
+
+<LanguageSection>
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+int t, n, k, h;
+int ans;
+int tmp;
+int limit;
```

Comment:
add extra lines above and below this

Reply:
these variables should be declared where they're used
just in general, you shouldn't declare all the variables at the start...